tests/test.bats

```diff
@@ -1,23 +1,8 @@
 #!/usr/bin/env bats
```

Comment:
Maybe we should stop testing with bats and delete all the related files.
You may make it a new issue.